velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2137,6 +2137,65 @@ TEST_F(DateTimeFunctionsTest, dateDiffTimestamp) {
           fromTimestampString("2018-02-28 10:00:00.500")));
 }
 
+TEST_F(DateTimeFunctionsTest, dateDiffTimestampWithTimezone) {
```

Comment:
This test file is getting super long. It would be nice to break it up into one file per function.

Reply:
I would suggest refactor in a separate PR.